azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/features/VirtualMachineApiLiveTest.java

```diff
@@ -96,6 +107,11 @@ public void setup() {
       nicName = nic.name();
 
       vmName = String.format("%3.24s", System.getProperty("user.name") + RAND + this.getClass().getSimpleName()).toLowerCase().substring(0, 15);
+
+      vaultResourceGroup = System.getProperty("test.vault.resource.group");
```

Comment:
I think this makes the test harder to execute: either you add it to `BaseAzureComputeApiLiveTest` or the test needs to create those prelimiary resources during the setup and destroy during tearDown. wdyt?